public/app/features/commandPalette/useMatches.ts

```diff
@@ -201,35 +201,27 @@ function useInternalMatches(filtered: ActionImpl[], search: string): Match[] {
         const haystackItem = haystack[haystackIndex];
 
         // Use the position of the match as a stand-in for score
-        const substringPosition = haystackItem.toLowerCase().indexOf(query);
+        const substringPosition = haystackItem.indexOf(query);
 
         if (substringPosition > -1) {
-          const score = haystack.length - substringPosition;
```

Comment:
@joshhunt  this looked like a mistake?

Reply:
yeah, it probably should have been `haystackItem.length - substringPosition` instead, to prioritise matches at the beginning of the string, but making substringPosition negative works as well.